docs/content/latest/architecture/docdb-replication/async-replication.md

```diff
@@ -73,30 +89,53 @@ Since 2DC replication is done asynchronously and by replicating the WAL (and the
+
+#### Bootstrapping sink clusters
+- Currently, it is the responsibility of the end user to ensure that a sink cluster has sufficiently recent updates so that replication can safely resume. In the future, bootstrapping the sink cluster can be automated. (TBD GH issue?)
+- Bootstrap currently relies on the underlying backup and restore (BAR) mechanics of YugabyteDB. This means it also inherits all of the limitations of BAR. For YSQL, currently the scope of BAR is at a database level, while the scope of replication is at table level. This implies that when bootstrapping a sink cluster, you will automatically bring in any tables from source database, in the sink database, even ones you might not plan to actually configure replication on. (TBD GH issue?)
+
+#### DDL changes
+- Currently, DDL changes are not automatically replicated. Applying commands such as CREATE TABLE, ALTER TABLE, CREATE INDEX to the sink clusters is the responsibility of the user.
+- DROP TABLE is not supported. The user must first disable replication for this table.
+- TRUNCATE TABLE is not supported. This is an underlying limitation, due to the level at which the two features operate, ie: replication is implemented on top of raft WAL files, while truncate is implemented on top of rocksdb SST files.
+- In the future, we will be able to propagate DDL changes safely to other clusters (TBD GH issue?).
+
+#### Safety of DDL and DML in active-active
+- Currently, certain potentially unsafe combinations of DDL/DML are allowed. For example, in having a unique key constraint on a column in an active-active last writer wins mode is unsafe since a violation could easily be introduced by inserting different values on the two clusters - each of these operations is legal in itself. The ensuing replication can, however, violate the unique key constraint. This will cause the two clusters to permanently diverge and the replication to fail.
+- In the future, allowing detection of such unsafe combinations and warn the user. Such combinations should possibly be disallowed by default. (TBD GH issue?)
+
+#### Scalability
+- When setting up replication, tables should be added in batches, until [#10611](https://github.com/yugabyte/yugabyte-db/issues/10611) is resolved.
+- When bootstrapping sink clusters, tables should be added in batches, until [#10065](https://github.com/yugabyte/yugabyte-db/issues/10065) is resolved.
+
+#### Kubernetes
+- Technically replication can be setup with kubernetes deployed universes. However, the source and sink must be able to communicate by directly referencing the pods in the other universe. In practice, this either means that the two universes must be part of the same kubernetes cluster, or that two kubernetes clusters must have DNS and routing properly setup amongst themselves. (TBH GH issue with more details?)
```

Comment:
I know of this issue https://github.com/yugabyte/yugabyte-db/issues/2422, but probably not what we are looking for here